README.md

```
@@ -1,97 +1,349 @@
+- `ESC` - Exit filter mode
+- `Enter` - Confirm filter / Execute task
+
+### Actions
+- `Enter` - Execute selected task
+- `v` - Toggle view mode
+- `?` or `F1` - Show help
+- `q` or `ESC` - Quit
+
+## 🎨 UI Preview
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)